bin/client.js

```diff
@@ -28,6 +28,7 @@ async function main() {
     const endpointURL = await discoverEndpointURL();
     url = `${endpointURL}/accept`;
   }
+  console.log("URL", url);
```

Comment:
I'm guessing this wasn't intentional.

Reply:
Not so much... though I suppose someday it might be nice to add a verbose option or something to that client to see what all the defaults end up being